tests/e2e/utils/pageNavigation.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios');
```

Comment:
same comment about possibly replacing **axios** w/ **got** to avoid multiple libraries that have similar functions. Or not, whatever.